app/questionnaire/value_source_resolver.py

```diff
@@ -162,6 +164,9 @@ def resolve(
     ) -> Union[ValueSourceEscapedTypes, ValueSourceTypes]:
         source = value_source["source"]
 
+        if not self.assess_routing_path:
+            self.assess_routing_path = source == "calculated_summary"
```

Comment:
It would be useful to add a comment as to which this is special.

Reply:
Added a comment now